en/docs/setup/configuring-a-read-write-active-directory-user-store.md

```diff
@@ -427,6 +433,35 @@ Default: not configured</td>
 <td>Retry the authentication request if a timeout happened
 <p>Default: not configured</p></td>
 </tr>
+<tr class="even">
+<td>UserIdSearchFilter</td>
+<td>user_id_search_filter</td>
+<td>UserID Search Filter</td>
+<td>Filtering criteria for searching a particular user entry.<br />
+Default : (&amp;(objectClass=person)(uid=?))</td>
+</tr>
+<tr class="odd">
+<td>java.naming.ldap.attributes.binary</td>
+<td>ldap_binary_attributes</td>
+<td>LDAP Binary Attributes</td>
+<td>Configure this to define the LDAP binary attributes separated by a space.
+<p>Default: not configured</p></td>
+</tr>
+<tr class="even">
+<td>ImmutableAttributes</td>
+<td>immutable_attributes</td>
+<td>Immutable Attributes</td>
+<td>Comma-separated list of user store maintained immutable attributes.
```

Comment:
How about "This is a comma-separated list of immutable attributes that are maintained in a user store."?